start-remote-agent-demo.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# set -x
+
+. /dev/stdin <<<"$(cat <(curl -s --raw https://raw.githubusercontent.com/bcgov/DITP-DevOps/main/code/snippets/getDockerHost))"
+export DOCKERHOST=$(getDockerHost)
+
+  # generate acapy plugin config file, writing $DOCKERHOST into URLs
+pushd ${SCRIPT_HOME}/aries-backchannels/acapy/ > /dev/null
```

Comment:
This line fails, as ${SCRIPT_HOME} is not defined.  If I just remove that and the “/“, everything works.  Alternatively, you could add the ${SCRIPT_HOME} calculation to the top of this.

Reply:
I'm so glad it worked! I'll fix that SCRIPT_HOME issue.

Reply:
Should be good to go now.